tests/lib/bin/mountinfo-tool

```diff
@@ -396,6 +396,29 @@ def renumber_devices(entry, seen):
     entry.dev = alloc_n(entry.dev)
 
 
+def renumber_ns(entry, seen):
+    # type: (MountInfoEntry, Dict[Tuple[Text, int], int]) -> None
+    """renumber_mount_ns re-numbers mount namespace ID from .root_dir property."""
+
+    def alloc_n(ns_type, ns_id):
+        # type: (Text, int) -> int
+        key = (ns_type, ns_id)
+        try:
+            return seen[key]
+        except KeyError:
+            n = len(seen)
+            seen[key] = n
+            return n
+
+    if entry.fs_type != "nsfs":
+        return
+    match = re.match(r"^([a-z_]+):\[(\d+)\]$", entry.root_dir)
```

Comment:
aren't we expecting `r"^mnt:\[(\d+)\]$"`?

Reply:
We only use the mount namespace but any namespace can be bound this way. The `fs_type` check makes sure we are catching only those things that are actual namespaces.

Reply:
Reading the kernel, it looks like we're only getting `mnt:[<inode>]` here, but I may be wrong.

Reply:
Whatever it is, it is not documented as anything stable. The inode is from nsfs which is obviously memory-only.

Reply:
What I meant by my reply to your initial question is that while the mount namespace uses `mnt:[...]` the other namespaces can be bound too and will yield their respective identifiers, so `pid:[...]` etc. I just did not want to special-case `mnt`.

Reply:
Yeah, let's keep the current code.